keyboards/pabile/p18/keymaps/default/keymap.c

```diff
@@ -0,0 +1,42 @@
+#include QMK_KEYBOARD_H
+/* 
+
+
+enable rgb here 
+
+
+*/
+
+
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+	[0] = LAYOUT(
+					KC_NO,KC_PMNS, KC_P9, KC_P8, KC_P7, 
+					KC_NO,KC_PPLS,KC_P6, KC_P5, KC_P4, 
+					KC_A, KC_TAB, KC_P3, KC_P2, KC_P1, 
+					KC_B, KC_PENT, KC_PDOT, KC_P0, KC_DEL)
+        
+/* alternate?
+	[0] = LAYOUT(
+					      KC_PMNS, KC_P9,   KC_P8, KC_P7, 
+					      KC_PPLS, KC_P6,   KC_P5, KC_P4, 
+					KC_A, KC_TAB,  KC_P3,   KC_P2, KC_P1, 
+					KC_B, KC_PENT, KC_PDOT, KC_P0, KC_DEL)
+*/
```

Comment:
```suggestion
```
As with the LAYOUT section of `p18.h`, I'd probably omit this without explicit inclusion of a `LAYOUT_alternate` or similar mechanism being present.